.github/workflows/ci.yaml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [ main ]
   pull_request:
-    branches: [ main ]
```

Comment:
Is this meant to be removed?

Reply:
yes, it doesn't hurt, it will run the CI on any PR even not targeting `main`

Reply:
oh i see, makes sense